src/main/io/osd.h

```diff
@@ -54,17 +52,23 @@ typedef struct {
     uint16_t alt_alarm;
 
     uint8_t video_system;
-    osd_unit_t units;
-} osd_profile_t;
+    uint8_t row_shiftdown;
```

Comment:
Argh. I was wondering if I should add this feature to driver level (max7456.c) or middle ware level (osd.c) and forgot about it...